.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,32 +4,30 @@ about: Suggest an idea for this project
 
 ---
 
-<!--
-PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
-
-ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
--->
+<!-- Please make sure you are posting an feature request pertaining to the Ionic Framework. -->
 
 # Feature Request
 
-**Ionic Info**
-Run `ionic info` from a terminal/cmd prompt and paste the output below.
+**Ionic version:**
+<!-- (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1) -->
+<!-- (For Ionic 2.x & 3.x issues, please use https://github.com/ionic-team/ionic-v3) -->
+[x] **4.x**
 
-```
-insert the output from ionic info here
-```
+**I'm submitting a ...**
+[ ] bug report
+[x] feature request

```

Comment:
Same with this, the heading tells us it's a feature request, as it is only used here: https://github.com/ionic-team/ionic/issues/new?template=feature_request.md